test-suite/non-functional-tests/test-scenario-6/config-template.properties

```diff
@@ -1,22 +1,21 @@
 # Values listed are default values which are also specified within the test plan
 
 # * Connection Properties *
-facadePort = {{facade-port}}
-facadeUrl = {{facade-url}}
-
-inboundUrl = {{inbound-url}}
-inboundPort = {{inbound-port}}
-
+facadePort = 8081
+facadeUrl = nia-gp2gp-elb-ebd5089784c3d1ec.elb.eu-west-2.amazonaws.com
```

Comment:
I think it'd be sensible to tear down the infrastructure once we're Done with this ticket, so I wouldn't commit the temporary URLs.

Reply:
ok, thanks, yes, we will do that...